test/specs/practise/PracticeCodeNFT.spec.js

```diff
@@ -1,69 +1,116 @@
 import * as dotenv from "dotenv";
```

Comment:
What is the purpose of the practice file and should it be skipped?

Reply:
The practice file is for practice purposes, So I initially write the code in the practice file and then implement it for each network as smoke and regression test cases.